### PR TITLE
#174797504 setup sequelize

### PR DESCRIPTION
**Pull request by valens10**
#### What does this PR do?
configure sequelize to be used as our ORM on the API.
#### Description of Task to be completed?
- install Sequelize and Sequelize-cli
- setup .sequeliceC file
- configure squelize
- Set up the migrations and models structure
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend.git`
- Switch your working directory `cd script-heroes-bn-backend`
- Checkout the branch `git checkout ch-setup-sequelize-#174797509`
- Create a .squelizeC file in the project root directory
- Configure the database/config.js file and fill out the required dtata
- Creating first Model (and Migration) with `sequelize model:generate --name User --attributes firstName:string,lastName:string,email:string`
- Running Migrations with `sequelize db:migrate`
- Creating First Seed with `sequelize seed:generate --name demo-user`
- Running Seeds with `sequelize db:seed:all `
- Switch to your database with this command `psql your_database_name`
- Run this query to make sure your data is in database `SELECT * FROM "Users"";`
#### What are the relevant pivotal tracker stories?
[#174797504](https://www.pivotaltracker.com/story/show/174797504)

### CODE REVIEW

**Review by dushimeemma — APPROVED** [review]
- Please fix some hound ci violations
- Write PR title as required

**Review by elvisiraguha — CHANGES_REQUESTED** [review]
@valens10 try to reduce these HoundCI warnings please.

**Comment by valens10** [comment]
> @valens10 try to reduce these HoundCI warnings please.

This is noted.

File: src/database/models/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+
+const config = require(__dirname + '/../config.js')[env];
```
[the remark this answers is not in the document]
Comment (Junior-Adadie) [reply src/database/models/index.js:5]: Try removing this .env code, the one working on env will add it in if necessary

File: src/database/models/item.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+'use strict';
```
[the remark this answers is not in the document]
Comment (Junior-Adadie) [reply src/database/models/item.js:1]: @valens10 Try removing the 'use strict' expression, I don't think it's required there

**Review by elvisiraguha — APPROVED** [review]

**Comment by kipropbrian** [comment]
Hey @valens10 you can ignore the database folder from hound violations and complaints, or find ways of fixing the complaints

**Review by Habinezajanvier — APPROVED** [review]
good

**Review by Adadie — APPROVED** [review]

**Comment by kipropbrian** [comment]
Some feedback. 
 - Sequelize and sequelize-cli should be in package.json 
- Your Readme has unnecessary things. It's like you copy pasted that from somewhere. Make it short and to the point. Too many stories do not help. 
- Remove your username from the configs. Do not use `valens` use `admin` or another name.
- sequelize-cli db:migrate does not work. 
- Make sure migrate and seed is working on travis

**Pull request merged**